linkml/generators/oocodegen.py

```diff
@@ -182,6 +186,9 @@ def create_documents(self) -> List[OODocument]:
                 elif range == "String":
                     default_value = '""'
 
+                # TODO:
```

Comment:
Needs wiring-up here! Not sure how the existing types need to be retained/overridden.